manual/source/utilities.rst

```diff
@@ -438,14 +445,29 @@ Language API: *Python*
 
     .. [#] write: https://github.com/cctbx/cctbx_project/blob/master/xfel/swissfel/jf16m_cxigeom2nexus.py
 
-* **h5py** (http://docs.h5py.org/)
+.. index:: API; Python; h5py
+
+* **h5py** [#]_
 	HDF5 for Python (h5py) is a general-purpose Python interface to HDF5.
 
+  .. [#] http://docs.h5py.org
+
+.. index:: API; Python; Mantis
+
 * **Mantis** [#]_, with NeXus-specific python code [#]_, reads :ref:`NXstxm`
 
   .. [#] Mantis: http://spectromicroscopy.com/
   .. [#] python code: https://bitbucket.org/mlerotic/spectromicroscopy/src/default/
 
+.. index:: API; Python; nexusformat
+
+* **nexusformat** [#]_ NeXus package for Python
+    Provides an API to open, create, and manipulate NeXus data.
```

Comment:
and plot

Reply:
That's a good addition for the nexpy/nexusformat project.